cmd/argo/commands/submit.go

```diff
@@ -111,5 +113,10 @@ func submitWorkflow(wf *wfv1.Workflow) error {
 		return err
 	}
 	printWorkflow(submitArgs.output, created)
+
+	if submitArgs.wait {
```

Comment:
I think this should be moved until after all workflows have been submitted. Otherwise it is submitting and waiting on them in sequence.

Reply:
Good catch! Fixed.